pyproject.toml

```diff
@@ -83,6 +85,9 @@ vllm_tgis_adapter = [
 
 [tool.pytest.ini_options]
 addopts = "-ra"
+markers = [
```

Comment:
Will also need to configure pytest to skip tests marked with large:

https://docs.pytest.org/en/7.1.x/example/markers.html#custom-marker-and-command-line-option-to-control-test-runs

Reply:
I've updated the marker to `hf_data` to capture under the assumption that it's not just about avoiding large model downloads but all HF data which can add up -- configured the test to exclude these